docs/06-modules/README.md

```diff
@@ -1,5 +1,7 @@
 # Kyma Modules
 
+> [!CAUTION] This is the open-source documentation for the Kyma project. If you are a managed offering user of **SAP BTP, Kyma runtime**, see [Kyma's Modular Approach](https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-s-modular-approach?version=Cloud) on SAP Help Portal.
```

Comment:
The note is not rendered correctly. I believe it should look like the one in the Quick Install doc. Is it because of a different indentation? Can't see it in the PR.
![Screenshot 2025-02-07 at 11 46 03](https://github.com/user-attachments/assets/fea76293-8532-4c5d-9a58-cce8ce783db6)